client.go

```diff
@@ -617,16 +620,24 @@ func (c *Client) connectToNode(node *Node) error {
 		}()
 	}
 
-	wsAddr := (&url.URL{Scheme: "ws", Host: node.Addr}).String()
-	dialer := &websocket.Dialer{
-		NetDialContext: c.config.WsDialContext,
-		Proxy:          http.ProxyFromEnvironment,
-	}
-	dialer.HandshakeTimeout = time.Duration(c.config.WsHandshakeTimeout) * time.Millisecond
+	var conn wsConn
+	var err error
+	if c.config.Webrtc {
+		c.peer.SetRemoteSdp(node.Sdp)
+		<-c.peer.OnOfferConnected
```

Comment:
we need to use a timeout to prevent this hanging forever. Maybe we can add a `c.config.WebRTCHandshakeTimeout` similar to `c.config.WsHandshakeTimeout`

Reply:
Yes, modified.